src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -139,6 +140,7 @@ public PublicClientApplicationBuilder WithIosKeychainSecurityGroup(string keycha
         }
 
         /// <summary>
+        /// For new applications, it is recommended to use the new WithBroker that accepts BrokerOptions as parameter

```

Comment:
Wouldn't it throw?

`WithBroker(bool)` and `WithBroker(BrokerOptions)` only works for UWP and Net6-Win. For all other targets, it will throw saying to use Msal.Broker package. Plus `WithBroker(BrokerOptions)` is not really applicable to UWP now?

Reply:
You are right.